docs/en/getting-started/example-datasets/uk-price-paid.md

```diff
@@ -346,67 +375,79 @@ ALTER TABLE uk_price_paid
     toYear(date) AS year,
     round(avg(price)) AS price,
     bar(price, 0, 1000000, 80)
 FROM uk_price_paid
 GROUP BY year
 ORDER BY year ASC;
+```
+
+Result:
 
+```text
 ┌─year─┬──price─┬─bar(round(avg(price)), 0, 1000000, 80)─┐
 │ 1995 │  67932 │ █████▍                                 │
 │ 1996 │  71505 │ █████▋                                 │
 │ 1997 │  78532 │ ██████▎                                │
-│ 1998 │  85435 │ ██████▋                                │
-│ 1999 │  96036 │ ███████▋                               │
-│ 2000 │ 107478 │ ████████▌                              │
-│ 2001 │ 118886 │ █████████▌                             │
-│ 2002 │ 137940 │ ███████████                            │
-│ 2003 │ 155888 │ ████████████▍                          │
+│ 1998 │  85436 │ ██████▋                                │
+│ 1999 │  96037 │ ███████▋                               │
+│ 2000 │ 107479 │ ████████▌                              │
+│ 2001 │ 118885 │ █████████▌                             │
+│ 2002 │ 137941 │ ███████████                            │
+│ 2003 │ 155889 │ ████████████▍                          │
 │ 2004 │ 178885 │ ██████████████▎                        │
-│ 2005 │ 189350 │ ███████████████▏                       │
+│ 2005 │ 189351 │ ███████████████▏                       │
 │ 2006 │ 203528 │ ████████████████▎                      │
-│ 2007 │ 219377 │ █████████████████▌                     │
+│ 2007 │ 219378 │ █████████████████▌                     │
 │ 2008 │ 217056 │ █████████████████▎                     │
 │ 2009 │ 213419 │ █████████████████                      │
-│ 2010 │ 236110 │ ██████████████████▊                    │
-│ 2011 │ 232804 │ ██████████████████▌                    │
-│ 2012 │ 238366 │ ███████████████████                    │
+│ 2010 │ 236109 │ ██████████████████▊                    │
+│ 2011 │ 232805 │ ██████████████████▌                    │
+│ 2012 │ 238367 │ ███████████████████                    │
 │ 2013 │ 256931 │ ████████████████████▌                  │
-│ 2014 │ 279917 │ ██████████████████████▍                │
-│ 2015 │ 297264 │ ███████████████████████▋               │
-│ 2016 │ 313197 │ █████████████████████████              │
-│ 2017 │ 346070 │ ███████████████████████████▋           │
-│ 2018 │ 350117 │ ████████████████████████████           │
-│ 2019 │ 351010 │ ████████████████████████████           │
-│ 2020 │ 368974 │ █████████████████████████████▌         │
-│ 2021 │ 384351 │ ██████████████████████████████▋        │
+│ 2014 │ 279915 │ ██████████████████████▍                │
+│ 2015 │ 297266 │ ███████████████████████▋               │
+│ 2016 │ 313201 │ █████████████████████████              │
+│ 2017 │ 346097 │ ███████████████████████████▋           │
+│ 2018 │ 350116 │ ████████████████████████████           │
+│ 2019 │ 351013 │ ████████████████████████████           │
+│ 2020 │ 369420 │ █████████████████████████████▌         │
+│ 2021 │ 386903 │ ██████████████████████████████▊        │
```

Comment:
Good, I see you've actually run these tutorials :)

Reply:
> Good, I see you've actually run these tutorials :)

Yeah, working with datasets is very engaging!